Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "polonius"
-version = "0.5.0"
+version = "0.8.0"
```

Comment:
Versions don't matter, but why 5 -> 8?

Reply:
so IIUC, they were apparently documented as released in the releases.md file, but not actually released

Reply:
I fixed the erroneous releases.md, and updated it to represent the current state of things, without missing an intermediate number, so this would be:
- `polonius-parser`: 0.4.0
- `polonius-engine`: 0.11.0
- `polonius`: 0.6.0

is that better ?